test/groovy/SeleniumExecuteTestsTest.groovy

```diff
@@ -39,6 +39,15 @@ class SeleniumExecuteTestsTest extends BasePiperTest {
 
     @Test
     void testExecuteSeleniumDefault() {
+        def expectedDefaultEnvVars = [
+            // following env vars are expected in JSON format
+            'PIPER_CONTAINER_PORT_MAPPING': '{"selenium/standalone-chrome":[{"containerPort":4444,"hostPort":4444}]}',
```

Comment:
@OliverNocon is the `hostPort` considered in any case or something we need to cleanup?